rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -771,7 +813,7 @@ namespace rsx
 					}
 					else
 					{
-						//allow_flush = false and not synchronized
+						//deferred_flush = true and not synchronized
```

Comment:
Space after `//`

Reply:
eh, we have a lot of comments all over that file that don't have a space after the `//`